b2b-app/imports/ui/admin/register/AboutForm.js

```diff
@@ -1,59 +0,0 @@
-import React from 'react'
```

Comment:
Just a question - why are we deleting these files - are they no longer needed?

Reply:
Every form is now dynamically rendered by Form.js (see 7443bf0) as it accepts a schema prop. All you need is the schema to fully specify it.